### ISSUE TITLE
Meaningless answer when running when running simple examples provided by the authors

### ISSUE THREAD

**Issue by vedernikovphoto** [InternLM/InternLM-XComposer#369]
![Screenshot 2024-07-11 at 17 51 34](https://github.com/InternLM/InternLM-XComposer/assets/51476414/2e8a07d5-4167-4f0b-bb39-6e3cc8310172)

My env:

Python 3.10.12

torch: 2.2.2+rocm5.6
torchvision: 0.17.2+rocm5.6
torchaudio: 2.2.2+rocm5.6

transformers: 4.33.1
timm: 0.4.12
sentencepiece: 0.1.99
gradio: 4.13.0
markdown2: 2.4.10
xlsxwriter: 3.1.2
einops: 0.7.0

**Comment by LightDXY** [comment]
it seems weird, is the image path correct?

**Comment by vedernikovphoto** [comment]
yes, the path is correct. I've downloaded the model and file from https://huggingface.co/internlm/internlm-xcomposer2d5-7b and try one of the examples proposed by author on their own image, haven't modified anything in the structure of the folders etc.

> it seems weird, is the image path correct?
